contracts/managementcontract/build_mgmt_contract.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+#
+# This script builds the guessing game from solidity to a go generated package
```

Comment:
@tudor-malene Is this comment a mistake? Should it refer to the management contract instead?

Reply:
copy pasta